content/source/docs/cloud/workspaces/managing-variables.html.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: "cloud"
+page_title: "Managing Variables - Terraform Cloud and Terraform Enterprise"
+description: "Configure Terraform input variables and environment variables and create reusable variable sets that apply to multiple workspaces."
+
+---
+
+# Managing Variables
+
+-> **Note:** Variable sets are in beta.
+
+You can set variables specifically for each workspace or you can create variable sets to reuse the same variables across multiple workspaces. Refer to the [variables overview](/docs/cloud/workspaces/variables.html) documentation for more information about variable types, scope, and precedence.
+
+You need [read and write variables permissions](/docs/cloud/users-teams-organizations/permissions.html#general-workspace-permissions) to create and edit both workspace-specific variables and variable sets through:
+
+- The Terraform Cloud UI, as detailed below.
+- The Variables API for [workspace-specific variables](/docs/cloud/api/workspace-variables.html) and [variable sets](/docs/cloud/api/variable-sets.html).
```

Comment:
`/docs/cloud/api/variable-sets.html` doesn't link appropriately when running locally. is this part of the `terraform` repo? if so, remember to ship them together!

Reply:
yes! Alex will merge those docs with this branch and then we'll do a ship all at once! thank you for the reminder! :-)